packages/doctor/README.md

```diff
@@ -1,6 +1,9 @@
-appium-doctor
-===================
-Attempts to diagnose and fix common Node, iOS and Android configuration issues before starting Appium.
+# @appium/doctor
+
+> Attempts to diagnose and fix common Appium configuration issues
```

Comment:
Reminder to myself - add a deprecation notice here, so people could use doctors scripts which are built in into drivers